src/std/io/socket/stream.ss

```diff
@@ -64,10 +64,7 @@
          (state-dir (direction->state dir)))
     (with-basic-socket-write-lock ssock
       (unless (stream-socket-closed? ssock state-dir)
-        (with-catch
-            (lambda (e)
-              (display "*** Warning: error shutting down socket: " (current-error-port))
-              (display-exception e (current-error-port)))
+        (with-catch void
```

Comment:
Intentional?

Reply:
Yes, this is intentional; if you are racing to close both directions you could end up getting a warning.
Also, this is a benign error I would say, it's very common for shutdown to fail, usually for non-error reasons (other side closed first).
So I removed the spam.